lib/Yam/Agama/Pom/GrubMenuAgamaOldEntryOrderPage.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles GRUB screen.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::GrubMenuAgamaOldEntryOrderPage;
```

Comment:
let's call it `GrubMenuAgamaDeprecatedEntryOrderPage` I think it is more accurate terms, please propagate the rename everywhere.